charts/dbildungscloud-iam-keycloak/values.yaml

```diff
@@ -1,11 +1,40 @@
-keycloakHostname: "keycloak.dev.spsh.dbildungsplattform.de"
```

Comment:
Changing the helm-value names here means that you also have to change it in the deploy pipeline in spsh-app-deploy. These variables (e.g. keycloakHostname, frontendHostname and backendHostname, db.name) are overwritten in the pipeline and must match.